reports/imports/import_2015.py

```diff
@@ -0,0 +1,36 @@
+from .base import BaseImport
+
+from reports.utils.canon import canon
+from reports.utils.work_sheet_mapper import mapper as work_sheet_mapper
+
+class Import2015(BaseImport):
+    """
+        Holds the methods that handles importing 2015 data files
+    """
+    def get_work_sheet(self, wb, old_sheet, year):
+        ws = None
+        for name in wb.sheetnames:
+            if name == work_sheet_mapper[old_sheet] or name.startswith(old_sheet + ' '):
+                ws = wb[name]
+                break
+        return ws
+
+    def import_1F(self, ws, sheet_info):
+        data = {}
+        self.slurp_table(ws, data, col_start=3, col_end=10, row_end=14)
+
+        return data
```

Comment:
QQ @ascii-dev ... Isn't this suppose to be just `1`? Does **2015** report have `1F` column that we should import data from?

Reply:
I decided to go with making the method names consistent because removing that will me more extra changes need to be made at the point where the methods are being called in the `import_from_file` method. With the way the implementation is currently, we've minimised the amount of changes that will be needed to extend the functions to support newer years. @kilemensi

Reply:
The `work_sheet_mapper` already deals with knowing which sheet mappings belongs to which year

Reply:
Yes but it doesn't make sense to go to 2015 import class and see import from sheets that do not exist @ascii-dev ... What is the point of separating 2015 import class from 2010 class if it's not clearly separate concerns?